src/main/java/duke/main/Parser.java

```diff
@@ -0,0 +1,52 @@
+package duke.main;
+
+import duke.DukeException;
+import duke.command.*;
```

Comment:
Any reason why separate classes for the commands are implemented instead of using an enum Command class?